dbt_schema_builder/builder.py

```diff
@@ -12,7 +12,8 @@
 from dbt.logger import GLOBAL_LOGGER as logger
 from dbt.logger import log_manager
 from dbt.task.compile import CompileTask
-from dbt.task.generate import _coerce_decimal, get_adapter
```

Comment:
This import is no more available in the newer version of `dbt_utils`

Reply:
nit: is there a reason not to have just added `from dbt.utils import _coerce_decimal` ?

Reply:
Yes that was necessary because [this function](https://github.com/dbt-labs/dbt-core/blob/v0.19.1/core/dbt/task/generate.py#L196) was no longer available in newer versions